### ISSUE TITLE
Using Frost-Server  without Semantic Sensor Network Ontology

### ISSUE THREAD

**Issue by claudiadragoste** [FraunhoferIOSB/FROST-Server#1947]
Hello everybody, 

I'm new in this so maybe is not the right place to put the question. I'm wondering if I can install and configure a Frost Server even if the database where I have the data is not structured using the Semantic Sensor Network Ontology (https://www.w3.org/TR/vocab-ssn/). My doubt is that Sensorthings Api needs to have the Semantic Sensor Network Ontology?

Thank you very much

**Comment by hylkevds** [comment]
The SensorThings API comes with its own data model, so you will have to map your current data model to the data model of the SensorThings API.
When it comes to the actual use of FROST, you can either use the database structure (tables) that FROST generates when you install it, and import your data into those tables (either through the API or directly into the tables), or you can create [database views](https://www.postgresql.org/docs/current/sql-createview.html) that mimic the FROST table structure, and directly use the data from your own tables through these views.

If you are not that interested in the SensorThings API, but want an OData interface on your current database, you can create JSON files that explain your data model to FROST, and then FROST can expose an OData interface directly on your tables.

**Comment by sgrellet** [comment]
and in addition with @hylkevds answer

tl;dr : SensorThings API/FROST will at the end of the day help you provide data compliant with the international standards in the observation/sample domain (ssn included): 

1°/ SensorThings API 1.1 data model is based on OGC/ISO 19156:2011 Observations & Measurements (O&M) + there is a strong O&M legacy in W3C:SSN

2°/ O&M revision ended last year into OGC/ISO 19156:2023 Observations, measurements & samples (OMS) and we currently work in archieving this target

![image](https://github.com/FraunhoferIOSB/FROST-Server/assets/16756304/74d07824-4618-4c6b-a5bc-c62c1714d411)

The expected timeline is the following

![image](https://github.com/FraunhoferIOSB/FROST-Server/assets/16756304/e32813f3-ae8c-40a6-8dd8-0365156976c9)

with this, we hope to be able at some point to even generate output compliant to SSN/SOSA from SensorThings API endpoints

**Comment by claudiadragoste** [comment]
Thank you very much for your time and your explanations. Thanks!
